addons/project/views/project_views.xml

```diff
@@ -374,7 +374,7 @@
                                     <div name="alias_def" colspan="2" attrs="{'invisible': [('alias_domain', '=', False)]}">
                                         <!-- Always display the whole alias in edit mode. It depends in read only -->
                                         <field name="alias_enabled" invisible="1"/>
-                                        <span class="font-weight-bold oe_read_only" attrs="{'invisible':  [('alias_name', '=', False)]}">Create tasks by sending an email to </span>
+                                        <span class="oe_read_only">Create tasks by sending an email to </span>
```

Comment:
I discussed with AJU about this fix, and she wants to:
- in readonly mode, 
    - display the text in bold if the alias_name is set.
    - display the text in muted otherwise.
- in edit mode, display the text in bold.

So we should adapt this fix like this:
```suggestion
                                        <span class="oe_read_only" attrs="{'invisible': [('alias_name', '!=', False)]}">Create tasks by sending an email to </span>
                                        <span class="oe_read_only" attrs="{'invisible': [('alias_name', '=', False)]}">Create tasks by sending an email to </span>
```

What do you think? 🙂

Reply:
Done.